proxy.go

```diff
@@ -598,12 +827,13 @@ func (proxy *proxy) init() error {
 	var l net.Listener
 	var err error
 
-	// flags
-	proxy.enableVMConsole = logrus.GetLevel() == logrus.DebugLevel
+	if !proxy.restoreState() {
+		// flags
+		proxy.enableVMConsole = logrus.GetLevel() == logrus.DebugLevel
 
-	// Open the proxy socket
-	if proxy.socketPath, err = getSocketPath(); err != nil {
-		return fmt.Errorf("couldn't get a rigth socket path: %v", err)
+		if proxy.socketPath, err = getSocketPath(); err != nil {
+			return fmt.Errorf("couldn't get a rigth socket path: %v", err)
```

Comment:
Typo that could be fixed now I've spotted it ;-) "rigth" => "right".

Reply:
Thanks, done.